src/App.js

```diff
@@ -10,6 +10,7 @@ import StorePage from "./pages/StorePage/StorePage";
 import WishlistPage from "./pages/WishlistPage/WishlistPage";
 import LandingPage from "./pages/LandingPage/LandingPage";
```

Comment:
Don't have to specify the page again in the component name, if it is already in the pages directory. 
Also consider importing them into one index file, and exporting that one file here instead

Reply:
Thanks for the suggestion, I'll do it.